exonum-java-binding-cryptocurrency-demo/frontend/src/proto/helpers.proto

```diff
@@ -0,0 +1,28 @@
+// Copyright 2018 The Exonum Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+import "google/protobuf/timestamp.proto";
+
+package exonum;
+
+message Hash { bytes data = 1; }
```

Comment:
Is there an easy way to reuse the same proto package in both core/binding/demo? As we have same messages defined in many places.

Reply:
Sorry, my mistake. I fixed.